cql/ManageRareAbnormality.cql

```diff
@@ -824,6 +852,96 @@ define CytologyInterpretedAsHsil:
       return aC in "HSIL"
   )  
 
+define CervixRemovalDateIsAfterMostRecentAisBiopsy:
+  (CervixRemovalDate is not null) and 
+  CervixRemovalDate after Collate.MostRecentAisBiopsy.date
+    
+define MostRecentAisBiopsy:
```

Comment:
It looks like this isn't actually used, and the one from `Collate` is used in throughout this file instead. To clean things up, it may help to either delete this expression here, or keep this expression and reference the one in this library throughout this file.